pkg/apiserver/apiserver.go

```diff
@@ -23,8 +23,8 @@ import (
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	"k8s.io/client-go/informers"
 
-	networkpolicyapis "github.com/vmware-tanzu/antrea/pkg/apis/networkpolicy"
-	"github.com/vmware-tanzu/antrea/pkg/apis/networkpolicy/install"
+	networkpolicyapis "github.com/vmware-tanzu/antrea/pkg/apis/networking"
```

Comment:
```suggestion
	networkingapis "github.com/vmware-tanzu/antrea/pkg/apis/networking"
```
or delete the alias as there is no package name conflict now.